manifests/plugin/container.pp

```diff
@@ -8,15 +8,23 @@
 class pulpcore::plugin::container (
   String $location_prefix = '/pulpcore_registry',
   String $registry_version_path = '/v2/',
+  String $registry_base_url = '',
 ) {
+
```

Comment:
Empty line here breaks linting in the future (once I get to updating `voxpupuli-test`)